src/main/java/com/maxmind/geoip2/model/AbstractResponse.java

```diff
@@ -5,6 +5,9 @@
 import com.fasterxml.jackson.databind.json.JsonMapper;
 import java.io.IOException;
 
+/**
+ * Abstract class for records that may be returned by the GeoIP2 web service.
```

Comment:
We use "record" somewhat particularly and distinguish it from these model classes. Maybe we could swap out that word. Also, this is used for both the web service and the databases.

```suggestion
 * Abstract class for GeoIP2 models.
```